posthog/management/commands/backfill_distinct_id_overrides.py

```diff
@@ -0,0 +1,125 @@
+from __future__ import annotations
+
+import logging
+from dataclasses import dataclass
+from typing import Iterator, NamedTuple
+
+import structlog
+from clickhouse_driver.errors import ErrorCodes, ServerException
+from django.core.management.base import BaseCommand, CommandError
+
+from posthog.clickhouse.client.execute import sync_execute
+from posthog.models.team.team import Team
+
+
+logger = structlog.get_logger(__name__)
+
+
+class Range(NamedTuple):
+    lower: int  # lower bound, inclusive
+    upper: int  # upper bound, exclusive
+
+    @property
+    def size(self):
+        return self.upper - self.lower
+
+    def split(self) -> Iterator[Range]:
```

Comment:
`Iterator[Range]` here is mostly for future proofing in case it makes sense to split into variable N values instead of a fixed N = 2.